tests/unit/test_adapter.py

```diff
@@ -578,11 +579,12 @@ def test__get_catalog_schemas(self):
         assert list(relations.values()) == [{"bar"}]
 
     @mock_athena
+    @mock_sts
     def test__get_data_catalog(self, aws_credentials):
         self.mock_aws_service.create_data_catalog()
         self.adapter.acquire_connection("dummy")
         res = self.adapter._get_data_catalog(DATA_CATALOG_NAME)
-        assert {"Name": "awsdatacatalog", "Type": "GLUE", "Parameters": {"catalog-id": "catalog_id"}} == res
+        assert {"Name": "awsdatacatalog", "Type": "GLUE", "Parameters": {"catalog-id": "123456789012"}} == res
```

Comment:
It should not change the value 🤔

Reply:
@Jrmyy if we have the default `awsdatacatalog` then catalog-id must be same as AWS Account ID. That's why we need `sts` lib as well

Reply:
Ok got it (the issue is that it is the same id as `SHARED_DATA_CATALOG_NAME` which confused me ...
Would it be possible to replace the value of `CATALOG_ID` by the value of `SHARED_DATA_CATALOG_NAME` (and indicates that this is the default value of moto and change the value of `SHARED_DATA_CATALOG_NAME` to an other value ?

Reply:
@Jrmyy I've fixed shared catalog tests, could you please check it again

Reply:
so, the moto's documentation is not actual :-) https://github.com/getmoto/moto/blob/e13c1f2be93dc49c256d36c5a4037f8ffb641184/moto/core/models.py#L28

Reply:
I've fixed it by using DEFAULT_ACCOUNT_ID constant from moto lib

Reply:
Ok grear 👌🏻